### ISSUE TITLE
The reproduce of 100 epochs

### ISSUE THREAD

**Issue by lzyhha** [bytedance/ibot#22]
Hello, first thanks for your great work.
Now, I want to reproduce the results of 100 epochs, i.e, the results of 71.5 knn in figure 8. 
Can you tell me the corresponding args?

**Comment by jensenzhoujh** [comment]
Hi @lzyhha ,

Check the model [here](https://drive.google.com/file/d/18ZX34jYhUcBQMgqqPhOtMnA8GQmM86nE/view?usp=sharing).

**Comment by lzyhha** [comment]
这是来自QQ邮箱的自动回复邮件。This is a automatic response.

**Comment by lzyhha** [comment]
Thanks for the provided checkpoint. And could you provide the args of this checkpoint?

**Comment by jensenzhoujh** [comment]
See 'args' in the checkpoint.

**Comment by lzyhha** [comment]
Ok, thanks for your reply! :blush:

**Comment by lzyhha** [comment]
Sorry, I cannot find args in the checkpoint. There are only ['student', 'teacher', 'optimizer', 'epoch', 'fp16_scaler', 'ibot_loss'].

**Comment by lzyhha** [comment]
> See 'args' in the checkpoint.

Sorry, I cannot find args in the checkpoint. There are only ['student', 'teacher', 'optimizer', 'epoch', 'fp16_scaler', 'ibot_loss'].

**Comment by jensenzhoujh** [comment]
Hi,

It uses the same args as the 800-epoch model. Except that we use the mask ratio of [0, 0.3] (throughout ablation) instead of [0, 0.3(+-0.2)]. Though there will be no big difference.

**Comment by zhangyikaii** [comment]
Maybe you need to change the `checkpoint_key` to an appropriate one? (like `state_dict`)

**Comment by lzyhha** [comment]
这是来自QQ邮箱的自动回复邮件。This is a automatic response.

**Comment by zhangyikaii** [comment]
Especially keys in `vitl_16_pt22k/checkpoint_student.pth` is different from the others, but overall, the code is very well written.

**Comment by lzyhha** [comment]
这是来自QQ邮箱的自动回复邮件。This is a automatic response.
